### ISSUE TITLE
Can't convergence for fine-tuning

### ISSUE THREAD

**Issue by oowe** [balancap/SSD-Tensorflow#132]
[The result show can't convergence for fine-tuning.]
https://ppt.cc/fq5MTx@.png

Hi all,

I have the problem is loss rate can't convergence.

I use the parameter are followed:

python3 train_ssd_network.py --train_dir=${TRAIN_DIR} --dataset_dir=${DATASET_DIR} --dataset_name=pascalvoc_2012 --dataset_split_name=train --model_name=ssd_300_vgg --checkpoint_path=${CHECKPOINT_PATH} --save_summaries_secs=60 --save_interval_secs=600 --weight_decay=0.0005 --optimizer=adam --learning_rate=0.0001 --batch_size=1

My environment:
cuda 8.0
cudnn6
opencv-python3.3
nvidia driver 375
tensorflow1.3

Does anyone know how can I do ?
thank!

**Comment by congjianting** [comment]
have you finded the solution?

**Comment by zoumengchao** [comment]
the same to me     up and down around 40,  while using caffe  the total loss <20  at begin

**Comment by zoumengchao** [comment]
mebe  the is some wrong in the code for calculate the total loss

**Comment by ivyteng** [comment]
Have you found the solution。 I encounter the same problem.

**Comment by 3born** [comment]
Have you tried evaluating with your model? In e.g. [issue 67](https://github.com/balancap/SSD-Tensorflow/issues/67) the observed total loss is high yet good results are achieved.
